Makefile

```diff
@@ -96,4 +100,4 @@ regenerate_test_proto:
 generate-test-local-certs:
 	mkcert localhost 127.0.0.1 ::1
 	mkcert -client localhost 127.0.0.1 ::1
-	cp (mkcert -CAROOT)/rootCA.pem test-certs
+	cp $(mkcert -CAROOT)/rootCA.pem test-certs
```

Comment:
This won't work for me 😄 Since I'm using `fish-shell`...

Reply:
Makefile:

`SHELL = /bin/sh` ^_^

Reply:
I guess we may use `$`, since most users use `bash`, `sh`, etc with this syntax.

Reply:
So let's merge ti as is with `$(mkcert -CAROOT)`?

Reply:
Yeah, sorry, was on a little vacation 😃